### PR TITLE
From `focalboard` to `board` to generate tar bundle

### PR DESCRIPTION
**Pull request by Rajat-Dabade**
Build name
Previous : `focalboard-<version>.tar.gz`
Now: `board-<version>.tar.gz`

### CODE REVIEW

**Review by M-ZubairAhmed — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -35,7 +35,8 @@ GO_BUILD_FLAGS ?= -ldflags '$(LDFLAGS)'
 MM_UTILITIES_DIR ?= ../mattermost-utilities
 DLV_DEBUG_PORT := 2346
 MATTERMOST_PLUGINS_PATH=$(MM_SERVER_PATH)/plugins
-FOCALBOARD_PLUGIN_PATH=$(MATTERMOST_PLUGINS_PATH)/focalboard
+BOARD_PLUGIN_PATH=$(MATTERMOST_PLUGINS_PATH)/board
+PLUGIN_NAME=board
```
Comment (marianunez) [inline Makefile:39]: Should be `Boards` plural

```suggestion
PLUGIN_NAME=boards
```

**Review by marianunez — CHANGES_REQUESTED** [review]

**Review by harshilsharma63 — APPROVED** [review]
Other than the comment Maria mentioned, that it should be plural `boards` instead of singular `board`, everything else looks good.

File: Makefile (hunk 1 above)
Comment (Rajat-Dabade) [reply Makefile]: Done.

File: Makefile [hunk 2]
```diff
@@ -35,7 +35,8 @@ GO_BUILD_FLAGS ?= -ldflags '$(LDFLAGS)'
 MM_UTILITIES_DIR ?= ../mattermost-utilities
 DLV_DEBUG_PORT := 2346
 MATTERMOST_PLUGINS_PATH=$(MM_SERVER_PATH)/plugins
-FOCALBOARD_PLUGIN_PATH=$(MATTERMOST_PLUGINS_PATH)/focalboard
+BOARD_PLUGIN_PATH=$(MATTERMOST_PLUGINS_PATH)/board
```
Comment (marianunez) [inline Makefile:38]:
```suggestion
BOARD_PLUGIN_PATH=$(MATTERMOST_PLUGINS_PATH)/boards
```

Comment (Rajat-Dabade) [reply Makefile]: Ah, missed it. Nice catch 🙌

**Review by marianunez — APPROVED** [review]

**Pull request merged**